runtime/bifrost/src/lib.rs

```diff
@@ -969,7 +969,7 @@ parameter_types! {
 	pub const SlotLength: BlockNumber = 8u32 as BlockNumber;
 	pub const XcmTransferOrigin: TransferOriginType = TransferOriginType::FromRelayChain;
 	pub XcmWeight: XcmBaseWeight = XCM_WEIGHT.into();
-	pub ContributionWeight:XcmBaseWeight = XCM_WEIGHT.into();
+	pub ContributionWeight:XcmBaseWeight = 893125000.into();
```

Comment:
What is this ContributionWeight for? And where does 893125000 come from? Can't this weight be covered by the 2 x XcmBaseWeight fee?
Besides, can you also elaborate what the follwoing item 1, 2, 3 represent?

I want to make sure we charge the right amount of transaction fee from the users. So please help to confirm the above information. Thanks!

![image](https://user-images.githubusercontent.com/45537372/132612996-04fa6478-a9c2-442a-92db-441db064547b.png)